### DISCUSSION TITLE
Sharing news about a label printer that works well! Zebra ZSB-DP14

### DISCUSSION THREAD

**Discussion by technigogo** [snipe/snipe-it#12284] [General]
I just bought a Zebra ZSB series label printer, and it works great with Snipe-IT. 

I am using the label cartridge ZSB-LC6, which is 2.25" x 1". I opted for the 4" model so we can print shipping labels, too, but the 2" model would also work perfectly. The 4" model runs about $200 at provantage and the 2" is $115. The labels are $22 for 1000 quantities, and they are permanent adhesive.

I can print from any browser, the settings are simple, and you just need to make sure to set the margins to "none" in the browser print dialogue the first time.

Here are the settings that work well in Snipe-IT.

![image](https://user-images.githubusercontent.com/38970081/209827713-5f31e6b6-26c0-49e6-8de1-5214a9e194c3.png)

**Comment by ghost** [comment]
Hey, how are you managing to get the labels to print in the correct orientation? I can't get them to print correctly onto LC106 Barcode labels.
